### DISCUSSION TITLE
I can't seem to login

### DISCUSSION THREAD

**Discussion by nammminn2** [Taxel/PlexTraktSync#1118] [Q&A]
i keep getting this error 
<img width="665" alt="CleanShot 2022-10-20 at 16 57 36@2x" src="https://user-images.githubusercontent.com/64177573/197056940-b91b8d7d-e211-4d96-bf75-5c2406ddc665.png">

**Comment by stevietv** [comment]
you are not using the right credentials then.
Does your plex account have 2FA and are you following the instructions then in blue?

**Reply by nammminn2** [reply]
> you are not using the right credentials then. Does your plex account have 2FA and are you following the instructions then in blue?

i don't have a 2fa and thats the correct password i used
